list.md

```diff
@@ -152,6 +152,12 @@ Bern University Library
 Universitätsbibliothek der Technischen Universität Wien
 [https://github.com/UBTUW](https://github.com/UBTUW)
 
+FID Judaica
```

Comment:
The [official name](http://sigel.staatsbibliothek-berlin.de/en/fid-kennzeichen/) is FID Jüdische Studien. Should we use that?